tests/foreman/cli/test_repository.py

```diff
@@ -642,7 +641,6 @@ def test_positive_update_checksum_type(self):
             u'content-type': content_type
         })
         self.assertEqual(repository['content-type'], content_type)
-        self.assertEqual(repository['checksum-type'], '')
```

Comment:
`checksum-type` is not returned if it wasn't set to any value, removing this line to unblock the test